test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -198,6 +198,21 @@ public async Task DotnetListPackage_InvalidFramework_Fail()
             }
         }
 
+        [Fact(Skip = "Requires a version of dotnet.exe that supports forwarding the --deprecated command argument.")]
```

Comment:
The test infrastructure seems to rely on a version of dotnet.exe that doesn't support forwarding the `--deprecated` command argument yet. Can't add/enable tests that rely on this argument until this is supported.